tools/ssh-bastion-ec2/ssh-bastion-ec2

```diff
@@ -163,27 +163,38 @@ def parse_arguments(argv):
                  help='Tag name to discover EC2 bastion hosts ')
   p.add_argument('-tv', '--tag-value', dest='tag_value', default="opensearch-bastion-hosts",
                  help='Tag value to discover EC2 bastion hosts for give tag name')
+  p.add_argument('-l', '--log', dest='log_level', default='WARN',
+                 help='Log level')
   args = p.parse_args(argv)
   return args
 
 
+def initialize_logging(log_level):
+  numeric_level = getattr(logging, log_level.upper(), None)
+  if not isinstance(numeric_level, int):
+    raise ValueError('Invalid log level: %s' % log_level)
+
+  logging.basicConfig(level=numeric_level, format='[%(levelname)s] %(message)s')
+
+
 def get_random_bastion_ip(tag_name, tag_value):
   ip_list = get_bastion_ips(tag_name, tag_value)
   return random.choice(ip_list) if ip_list else None
 
 
 def __main__(argv):
   args = parse_arguments(argv)
+  initialize_logging(args.log_level)
   bastion_ip = get_random_bastion_ip(args.tag_name, args.tag_value)
   if not bastion_ip:
-    print("No bastion host IPs found. Exiting!")
+    logger.error("No bastion host IPs found. Exiting!")
```

Comment:
I don't see the exiting happening here. Am I missing something?

Reply:
Add appropriate exits.